### PR TITLE
Add log out button & validate token for Users

### PR DESCRIPTION
**Pull request by LucasKnudsen**
[**PT-Story:**  ](https://www.pivotaltracker.com/story/show/178214061)
### User Story 
``` 
As a logged in user
In order to have a more seamless authentication
I want the site to remember be, but also be in control of if I want to be remembered 
``` 
## Changes proposed in this pull request: 
* Adds logout button that 1) clears localStorage, 2) sets application state, 3) destroys the session
* Adds continuous token validation

## What I have learned working on this feature: 
* Logging out

### CODE REVIEW

File: src/components/layout/Navbar.jsx [hunk 1]
```diff
@@ -22,16 +26,30 @@ const Navbar = () => {
...
+                active
+                as={Link}
+                to='/login'
+              />
+            )}
           </Menu.Menu>
         </Menu>
       </Segment>
+      <div>
+
```
Comment (offcast-dev) [inline src/components/layout/Navbar.jsx]: empty div?

**Review by DavDan1 — APPROVED** [review]

File: src/modules/Authentication.js [hunk 2]
```diff
@@ -53,6 +71,17 @@ const Authentication = {
     }
     setLoading(false);
   },
+
+  async validateToken() {
+    axios
+      .get('/auth/validate_token', { headers: getUserAuthToken() })
+      .then((response) => {
+        debugger;
```
Comment (offcast-dev) [inline src/modules/Authentication.js]: oops

**Review by offcast-dev — CHANGES_REQUESTED** [review]
remove the debugger))

File: src/components/layout/Navbar.jsx (hunk 1 above)
Comment (LucasKnudsen) [reply src/components/layout/Navbar.jsx]: For semantic purposes 😏

File: src/modules/Authentication.js (hunk 2 above)
Comment (LucasKnudsen) [reply src/modules/Authentication.js]: 💯

**Review by offcast-dev — APPROVED** [review]

**Review by 1CIM — APPROVED** [review]

**Review by tochman — APPROVED** [review]
Goog job

**Pull request merged**